h2o-algos/src/test/java/hex/glm/GLMMojoWithOffsetTest.java

```diff
@@ -0,0 +1,208 @@
+            params._offset_column = "offset";
+            GLMModel model = new GLM(params).trainModel().get();
+            Scope.track_generic(model);
+            Frame scoreFrame = model.score(prostateTrain);
+            Scope.track(scoreFrame);
+            
+            Model.JavaScoringOptions options = new Model.JavaScoringOptions();
+            options._abs_epsilon = 1e-6;
+            options._fraction = 1.0;    // check all rows
+            options._disable_pojo = true;   // only test mojo, no offset support for pojo
+            Assert.assertTrue(model.testJavaScoring(prostateTrain, scoreFrame, 1e-6, options));
+            
+            // model without offset should be different
+            params._offset_column = null;
+            params._ignored_columns = new String[]{"offset"};
+            GLMModel model2 = new GLM(params).trainModel().get();
+            Scope.track_generic(model2);
+            Frame scoreFrame2 = model2.score(prostateTrain);
+            Scope.track(scoreFrame2);
+            Assert.assertTrue(Math.abs(scoreFrame.vec(0).at(0)-scoreFrame2.vec(0).at(0))>1e-3);
+        } finally {
+            Scope.exit();
+        }
+    }
+    
+    @Test
+    public void testOffsetBinomial() {
+        Scope.enter();
+        try {
+            Frame prostateTrain = parseTestFile("smalldata/glm_test/prostate_cat_train.csv");
+            Frame trainOffset = TestUtil.generateRealOnly(1, (int) prostateTrain.numRows(), 0);
+            trainOffset.setNames(new String[]{"offset"});
+            DKV.put(trainOffset);
+            Scope.track(trainOffset);
+            prostateTrain.add(trainOffset);
+            DKV.put(prostateTrain);
+            Scope.track(prostateTrain);
+
+            Frame prostateTest =  parseTestFile("smalldata/glm_test/prostate_cat_test.csv");
+            Frame testOffset = TestUtil.generateRealOnly(1, (int) prostateTest.numRows(), 0);
+            testOffset.setNames(new String[]{"offset"});
+            DKV.put(testOffset);
+            Scope.track(testOffset);
+            prostateTest.add(testOffset);
+            DKV.put(prostateTest);
+            Scope.track(prostateTest);
+
+            GLMModel.GLMParameters params = new GLMModel.GLMParameters(GLMModel.GLMParameters.Family.binomial);
+            params._response_column = "CAPSULE";
+            params._ignored_columns = new String[]{"ID", "RACE", "DPROS", "DCAPS"};
+            params._train = prostateTrain._key;
+            params._offset_column = "offset";
+            GLMModel model = new GLM(params).trainModel().get();
+            Scope.track_generic(model);
+            Frame scoreTest = model.score(prostateTest);
+            Scope.track(scoreTest);
+            Model.JavaScoringOptions options = new Model.JavaScoringOptions();
+            options._abs_epsilon = 1e-6;
+            options._fraction = 1.0;    // check all rows
+            options._disable_pojo = true;   // only test mojo, no offset support for pojo
```

Comment:
what happens when user tries to download POJO for model trained with offset column?

it seems like havePOJO function should be modified

Reply:
My guess is nothing will happen.

Reply:
I added the check to make no pojo if user specify offsetColumn